dotnet-datetime/DateTimeOffsetVsDateTime/DateTimeOffsetVsDateTime/Program.cs

```diff
@@ -0,0 +1,23 @@
+﻿// See https://aka.ms/new-console-template for more information
+using System.Diagnostics;
+
+var dateTime = DateTime.Now;
+Console.WriteLine($"DateTime: {dateTime}"); 
```

Comment:
Can we add some spacing here between each of the code snippets as it's very hard to read as it is currently formatted. It should reflect each of the sections in the article.